Ray

sch7878 opened the pull request.

sch7878 committed:
[update]자외선 알아보기

sch7878 committed:
[update]ì자외선 알아보기

추가 문구가 뜨게 수정

hyeunseungYu reviewed (changes requested):
uvStatus에 '위험' 추가해주시면 좋을 것 같습니다!

sch7878 committed:
[update]자외선 확인하기

>자외선 상태 위험 ì가 및 낮음,보통에 따른 추가 문구 작성, id값 변경

sch7878 committed:
[update]자외선 확인하기

오타수정

hyeunseungYu reviewed (approved):
LGTM😀

dirn0568 commented:
확인했습니다

jung2232 reviewed (commented):
good

dirn0568 reviewed (commented):
고생하셨습니다

dirn0568 reviewed (approved):
고생하셨습니다

hyeunseungYu committed:
[merge] ray branch conflict 처리

CONFLICT (add/add): Merge conflict in .idea/.gitignore
Auto-merging .idea/misc.xml
CONFLICT (add/add): Merge conflict in .idea/misc.xml
Auto-merging .idea/modules.xml
CONFLICT (add/add): Merge conflict in .idea/modules.xml
CONFLICT (modify/delete): README.md deleted in HEAD and modified in main.  Version main of README.md left in tree.ee.
Auto-merging app.py
CONFLICT (content): Merge conflict in app.py
Auto-merging templates/index.html
CONFLICT (content): Merge conflict in templates/index.html
Automatic merge failed; fix conflicts and then commit the result.
